guides/release/tutorial/service.md

```diff
@@ -87,19 +87,19 @@ otherwise we will create a new HTML element and call our Leaflet map service to
 import { camelize } from '@ember/string';
 import Service from '@ember/service';
 import { set } from '@ember/object';
-import { inject as service } from '@ember/service';
+import { inject as service } from '@ember-decorators/service';
 
-export default Service.extend({
+export default class MapElementService extends Service {
 
-  geocode: service(),
-  map: service(),
+  @service geocode;
+  @service map;
 
-  init() {
+  constructor() {
     if (!this.cachedMaps) {
       set(this, 'cachedMaps', {});
     }
-    this._super(...arguments);
-  },
+    super();
```

Comment:
Works without passing constructor arguments (presently), but recommend passing them along in case something changes in the future.
```suggestion
    super(...arguments);
```